at-luantran/Week4/Human_Resources/models/employee.rb

```diff
@@ -0,0 +1,30 @@
+require 'bcrypt'
+
+class Employee
+  attr_accessor :id, :name, :team, :role, :username, :password, :leave_balance
+  def initialize; end
+
+  def self.check_login(username, password)
+    sql = "SELECT * FROM `LuanTranHumanResources`.`employee` WHERE username = '#{username}'"
+    result = $connect.query(sql)
+    result.each do |x|
+      password == BCrypt::Password.new(x.values_at('password').first)
+    end
+  end
+
+  def self.add_employee(name, username, password)
+    sql = "INSERT INTO `LuanTranHumanResources`.`employee`(name, role, username, password, leave_balance)
+                        VALUES('#{name}', 'Employee', '#{username}', '#{password}', 10)"
+    $connect.query(sql)
+  end
+
+  def self.em_forgot(username)
+    sql = "SELECT * FROM `LuanTranHumanResources`.`employee` WHERE username = '#{username}'"
+    $connect.query(sql)
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/d3845af1984318eb2e065c76b954d85f9dee6230/at-luantran/Week4/Human_Resources/models/employee.rb#L23"></span>Style/GlobalVars: Do not introduce global variables. (https://github.com/rubocop-hq/ruby-style-guide#instance-vars, http://www.zenspider.com/Languages/Ruby/QuickRef.html)</td>
    </tr>
  </tbody>
</table>